### DISCUSSION TITLE
Support `interfaceName` for all field types

### DISCUSSION THREAD

**Discussion by ssyberg** [payloadcms/payload#3143] [Feature Requests & Ideas]
In our project we have a large set of reusable field configurations and we recently noticed the generated GraphQL enums for one of our `select` fields was forcing us to create a ton of fragment alias' despite the fact that they all pointed to the exact same select field configuration (and hence select options). I believe adding `interfaceName` for other field types would solve this?

**Comment by kariyngva** [comment]
As a workaround I've sometimes resulted to wrapping `select` in a `group` and setting `interfaceName` on the group.
